### PR TITLE
Shoe classification

### PR DESCRIPTION
**Pull request by adithya-s-k**
# Pull Request for DL-Simplified 💡

## Issue Title : Shoes Classification #37

- **Info about the related issue (Aim of the project)** : Create a model to identify the brands of the shoes.
- **Name:**Adithya S Kolavi
- **GitHub ID:** adithya-s-k
- **Email ID:** adithyaskolavi@gmail.com
- **Idenitfy yourself:  `SSOC Participant`

Closes: #37 

### Describe the add-ons or changes you've made 📃

Shoe Classification using SVM and CNN

## Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested? ⚙️

Describe how it has been tested
I have tested the model out using the testing data set and also have put the accuracy of it on the readme file

## Checklist: ☑️

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
1. You don't have to upload all the images of the test set, train set and validation set. Just add a `README.md` file in the `Dataset` folder and inside which put the URL of the dataset that has been used here. Remove all those extra images.
2. You need to create a single `.ipynb` file and inside which you need to put the whole project altogether. All the models built by you, will be inside a single `.ipynb` file.

@adithya-s-k

**Comment by adithya-s-k** [comment]
@abhisheks008 I have made all the necessary changes please look into it

**Review by abhisheks008 — APPROVED** [review]
Your changes have been approved. You have done a great work for this project, keep going. @adithya-s-k
